### PR TITLE
🐛 fix: prefix dependabot commit message with :arrow_up: gitmoji

### LINKED ISSUE #22: Dependabot PRs don't adhere to PR title commitlint config
#10 + #19 -> a clash! 💥 

Copying from https://github.com/LearningTypeScript/projects/pull/20#issuecomment-1133971989: looks like dependabot doesn't pick up on commitlint configurations. I guess that has to be configured. https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#commit-message

### PR DESCRIPTION
**Pull request by fuadop**
## PR Checklist

-   [x] Addresses an existing open issue: fixes #22 
-   [x] That issue was marked as [accepting prs](https://github.com/LearningTypeScript/projects/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
-   [x] Steps in [CONTRIBUTING.md](https://github.com/LearningTypescript/projects/blob/main/CONTRIBUTING.md) were taken

## Overview
I hope this fixes it @JoshuaKGoldberg .

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -9,4 +9,7 @@ updates:
     target-branch: main
     labels:
       - "npm"
-      - "dependencies"
\ No newline at end of file
+      - "dependencies"
+    commit-message:
+      prefix: "⬆️ "
```
Comment (JoshuaKGoldberg) [inline .github/dependabot.yml:14]: Hmm, when I run this locally, I get a complaint:

```shell
~/repos/learning-typescript-projects$ echo "⬆️ build(deps-dev): bump ts-jest from 28.0.0 to 28.0.2" | npx commitlint
⧗   input: ⬆️ build(deps-dev): bump ts-jest from 28.0.0 to 28.0.2
✖   Your commit should start with gitmoji code,please check the emoji code on https://gitmoji.dev/. [start-with-gitmoji]
✖   subject may not be empty [subject-empty]
✖   type may not be empty [type-empty]

✖   found 3 problems, 0 warnings
ⓘ   Get help: https://github.com/conventional-changelog/commitlint/#what-is-commitlint
```

Do you?

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
This might not work? What does your local system say?

#24 is of note too.

File: .github/dependabot.yml (hunk 1 above)
Comment (fuadop) [reply .github/dependabot.yml:14]: I don't get this but, let me push a change in another format.. let me write it in gitmoji format instead of emoji.

Comment (fuadop) [reply .github/dependabot.yml]: Can you try it now? I used gitmoji instead of emoji now.

**Review by JoshuaKGoldberg — APPROVED** [review]
Let's try it out!

File: .github/dependabot.yml (hunk 1 above)
Comment (JoshuaKGoldberg) [reply .github/dependabot.yml]: `echo ":arrow_up: build(deps-dev): bump abc to 123" | npx commitlint` works for me! 🚀 

I can't find a way to get Dependabot to create a _new_ pull request (do you know of one?), so I guess we have to ... wait until the next update comes in

Comment (fuadop) [reply .github/dependabot.yml]: Yea we have to wait till another dependency update 😅

**Pull request merged**